test/unittests/mongoBackend/mongoUpdateContext_withOnchangeSubscriptions_test.cpp

```diff
@@ -3087,13 +2948,11 @@ TEST(mongoUpdateContext_withOnchangeSubscriptions, DISABLED_MongoDbQueryFail)
     BSONObj sub = connection->findOne(SUBSCRIBECONTEXT_COLL, BSON("_id" << OID("51307b66f481db11bf860001")));
     EXPECT_EQ(20000000, sub.getIntField("lastNotification"));
 
-    /* Release connection */
-    setMongoConnectionForUnitTest(NULL);
-
+    /* Restore real DB connection */
+    setMongoConnectionForUnitTest(connectionDb);
```

Comment:
I was expecting to see a *setMongoConnectionForUnitTest(connectionDb);* in utExit ...
Not sure I understand why "Restore real DB connection" is done only sometimes and not in all test cases ...